builder/ecs/step_stop_builder.go

```diff
@@ -0,0 +1,25 @@
+package ecs
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+)
+
+type stepStopBuilder struct {
```

Comment:
I see what this is used for. But it was a little confusing given the fact that we still append the image creation steps. I left a suggestion above that removes the need for this step. Let me know what you think.

Reply:
I created this file because I want to use this file again if there are other parameters that need to skip building the image. I think it is better to remove this step too.